__backend_test__/packagesEndpoint.test.ts

```diff
@@ -1,6 +1,6 @@
 import { Request, Response } from 'express';
 import * as prismaCalls from '../backend/prismaCalls';
```

Comment:
I named the file wrong but intention is to have a separate file for each endpoint. So make a separate file for each of the 2 endpoints you have or else the file is going to get extremely large at the end of this.